homeworks/ekaterina.sharova_rogerosha/PokemonProject/pokemons.css

```diff
@@ -0,0 +1,152 @@
+* {
+  box-sizing: border-box;
+}
+
+.upper-header {
+  background-color: #333;
+  display: grid;
+  align-items: center;
+}
+
+.ma-logo {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
+
+.MAlogo {
+  width: 29.11px;
+  height: 29.95px;
+  margin-left: 53px;
+}
+
+.MA {
+  width: 147.68px;
```

Comment:
please remove numbers after `.` everywhere in your css files for `px` sizes

Reply:
Thank you very much for this big review of my homework!